### DISCUSSION TITLE
Remix Run support?

### DISCUSSION THREAD

**Discussion by nghiepdev** [urql-graphql/urql#2267] [Q&A]
Hey team,

Are there plans to support Remix Run?

**Comment by mocon** [comment]
I’m using `graphql-request` with GraphQL code generator, which allows us to load our typed queries in `loader`, and perform mutations in `action`. This works well for everything except subscriptions, where I’m adding `urql` on client-side only. I’d be curious if the team has any suggestions for this though!

**Reply by frederikhors** [reply]
And what about the normalized cache in browser? What are you using?

**Comment by nghiepdev** [comment]
Sorry mentioned @kitten @JoviDeCroock, 
Are there plans to support Remix?

**Reply by JoviDeCroock** [reply]
Remix is already supported as we are supporting React 😅
